src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/phpunit-bridge",
+    "type": "symfony-bridge",
+    "description": "Symfony PHPUnit Bridge",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
```

Comment:
Not sure what approach we are following here. But Form and PropertyAccess Component for example created by Bernhard have Fabien as author. So it's kinda inconsistent. I guess we should either only add the Symfony Community, or maybe all Mergers to the author. This way it also helps as reference to find a contact person.